parrots.json

```diff
@@ -455,5 +455,9 @@
     {
       "hd": "hd/transparront.gif",
       "name": "Transparront"
-    }
+    },
+    {
+      "gif": "fixparrot.gif",
+      "name": "Fix Parrot"
+    },
```

Comment:
Looks like an extra comma snuck in there, mind removing that?

Reply:
Sure!